### PR TITLE
feat: JSR support for Cloudflare SDK.

### PR DESCRIPTION
**Pull request by yusinto**

### CODE REVIEW

File: packages/sdk/cloudflare/src/createPlatformInfo.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import type { Info, PlatformData, SdkData } from '@launchdarkly/js-server-sdk-common-edge';
 
-import { name, version } from '../package.json';
+// @ts-ignore
+// eslint-disable-next-line prettier/prettier
+import * as packageJson from '../package.json' assert { type: "json" }
```
Comment (yusinto) [inline packages/sdk/cloudflare/src/createPlatformInfo.ts:5]: This import assertion is new-ish and is not yet fully supported by jest and prettier. I took a shortcut here and ignore this for now to save time. Hopefully in time we can just remove these ignores.

File: .prettierignore [hunk 2]
```diff
@@ -10,3 +10,4 @@ coverage
 .vscode
 **/*/CHANGELOG.md
 packages/sdk/akamai-edgekv/src/edgekv/edgekv.js
+packages/sdk/cloudflare/src/createPlatformInfo.ts
```
Comment (yusinto) [inline .prettierignore:13]: `@trivago/prettier-plugin-sort-imports` doesn't like the new-ish import assertion syntax. See my comments [here](https://github.com/launchdarkly/js-core/pull/415/files#r1546864891).

File: packages/sdk/cloudflare/example/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   "devDependencies": {
     "@cloudflare/workers-types": "^4.20230321.0",
     "@types/jest": "^29.5.5",
-    "esbuild": "^0.14.41",
+    "esbuild": "0.20.2",
```
Comment (yusinto) [inline packages/sdk/cloudflare/example/package.json:13]: Use the latest version of esbuild to get import assertion support.

File: packages/sdk/cloudflare/example/src/index.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { init as initLD } from '@launchdarkly/cloudflare-server-sdk';
 
 export default {
   async fetch(request: Request, env: Bindings, ctx: ExecutionContext): Promise<Response> {
-    const clientSideID = 'test-client-side-id';
+    const clientSideID = 'test-sdk-key';
```
Comment (yusinto) [inline packages/sdk/cloudflare/example/src/index.ts:6]: This was inconsistent with the README so it's kinda like a bugfix.

File: packages/sdk/cloudflare/example/package.json (hunk 3 above)
Comment (kinyoklion) [inline packages/sdk/cloudflare/example/package.json:13]: Is this intentionally pinned?

File: packages/sdk/cloudflare/jsr.json
Comment (yusinto) [inline packages/sdk/cloudflare/jsr.json:1]: This is the jsr equivalent of package.json. It's used during publishing. The version is hardcoded atm, but I'll submit a separate PR to implement publishing through release-please.

File: packages/sdk/cloudflare/src/index.ts
Comment (yusinto) [inline packages/sdk/cloudflare/src/index.ts:1]: JSR publish command complained about these so I corrected them.

File: packages/sdk/cloudflare/example/package.json (hunk 3 above)
Comment (yusinto) [reply packages/sdk/cloudflare/example/package.json]: I've carot'd it.

**Review by kinyoklion — CHANGES_REQUESTED** [review]

File: packages/sdk/cloudflare/jsr.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@launchdarkly/cloudflare-server-sdk",
+  "version": "0.0.3",
```
[the remark this answers is not in the document]
Comment (yusinto) [reply packages/sdk/cloudflare/jsr.json:3]: Yes you are right. I was going to submit a separate PR for the RP work. Maybe i'll just do it together in this PR now that you mentioned it.

**Review by kinyoklion — APPROVED** [review]

File: packages/sdk/cloudflare/example/README.md [hunk 6]
```diff
@@ -38,7 +37,14 @@ npx wrangler kv:key put --binding=LD_KV "LD-Env-test-sdk-key" --path ./src/testD
 npx wrangler kv:key get --binding=LD_KV "LD-Env-test-sdk-key" --preview
 ```
 
-5. Finally:
+5. Edit [index.ts](https://github.com/launchdarkly/js-core/blob/main/packages/sdk/cloudflare/example/src/index.ts#L6) to use your clientSideID and a valid flag key from the test data you just inserted.
+
+```ts
+    const clientSideID = 'test-sdk-key';
```
Comment (louis-launchdarkly) [inline packages/sdk/cloudflare/example/README.md:43]: My assumption is that there is clear documentation somewhere explaining this (as SDK Key is the terminology we typically use for Server-Side) - we need the Client-side ID for the events here, is that correct?

Comment (yusinto) [reply packages/sdk/cloudflare/example/README.md:43]: We need the clientSideID yes. I will correct them all to clientSideID to be consistent.

**Review by louis-launchdarkly — APPROVED** [review]

File: packages/sdk/cloudflare/jsr.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (yusinto) [reply packages/sdk/cloudflare/jsr.json:3]: I will do this in a separate pr focussed on publishing in CI.

**Pull request merged**